Source/OIDRegistrationRequest.h

```diff
@@ -79,7 +85,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)init NS_UNAVAILABLE;
 
-/*! @brief Designated initializer.
+/*! @brief Initializer for source compability with older versions
```

Comment:
Is the intention to deprecate this method? If so it should be marked deprecated, if not then this wording should be changed.

Reply:
I'm not sure what the AppAuth policy on deprecations / source compatibility is. I guess perhaps deprecation should only happen at a major version release and actual API removal the following major version?

I've updated the text to say ' Initializer if no initial access token required'. I'm equally happy to mark it deprecated if you think that's better, or change it to a better wording. I'm 50:50.